corehq/toggles.py

```diff
@@ -1785,7 +1785,7 @@ def _commtrackify(domain_name, toggle_is_enabled):
 
 MANAGE_CCZ_HOSTING = StaticToggle(
     'manage_ccz_hosting',
-    'Allow project to configure ccz hosting',
+    'Allow project to manage ccz hosting',
```

Comment:
tendency is to search for "manage" on the feature flag listing and hence this was not appearing as a result.